queries.sql

```diff
@@ -7,4 +7,13 @@ SELECT (name, escape_attempts) FROM animals WHERE weight_kg > 10.5;
 SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+DELETE FROM animals WHERE date_of_birth < '01-Jan-2020';
+UPDATE animals SET weight_kg = weight_kg * -1;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
```

Comment:
- [ ] Kindly make sure to add  all queries for all the transactions.
Currently, you have only added the last transaction and also its not complete.
Transaction starts with 
`BEGIN;`
Example: (for the code you wrote)-
```sql
BEGIN;

    DELETE FROM animals WHERE  date_of_birth >= '2022/01/01' ;
    SAVEPOINT SP1;

    UPDATE animals SET weight_kg = weight_kg * -1; 
    ROLLBACK TO SP1;
    UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0; 

    COMMIT;
```